CMakeLists.txt

```diff
@@ -45,12 +45,13 @@ target_include_directories(${PROJECT_NAME} INTERFACE
 )
 
 # Install Package Configuration
-install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}_targets)
-install(EXPORT ${PROJECT_NAME}_targets
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
+install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME_LOWER}_targets)
+install(EXPORT ${PROJECT_NAME_LOWER}_targets
     NAMESPACE ${PROJECT_NAME}::
-    FILE ${PROJECT_NAME}-config.cmake
+    FILE ${PROJECT_NAME_LOWER}Config.cmake

```

Comment:
Per your description, should this be `${PROJECT_NAME_LOWER}-config.cmake` OR `${PROJECT_NAME}Config.cmake`? This looks like a mix of the two that probably works, but isn't either of what you found.

Reply:
What is your desired usage:
`find_package(WIL)`
or 
`find_package(wil)`
or even
`find_package(Wil)`
?

Reply:
Ping @jonwis for response.

Reply:
Ping again.